titus-server-master/src/main/java/com/netflix/titus/master/kubernetes/pod/KubePodConfiguration.java

```diff
@@ -135,6 +135,19 @@
     @DefaultValue("NONE")
     String getDisabledJobSpreadingPattern();
 
+    /**
+     * Job spreading skew adjustment factor, with the skew computed as job_size/alpha. Setting value >= 1 effectively
```

Comment:
Do you mean `<` 1?  We expect to set this value greater than 1.  In fact the default is `3`.  I prefer naming this variable as `desiredNodeCount` or something like that, or at least comment this as the motivation for the value, so in 6 months we don't have to figure out what "alpha" means again.

Reply:
Yes, I meant `<`. Will fix it.
I did not want to use the term `desiredNodeCount` as this is true only for small jobs. It is more like a number of job partitions, where each partition may be placed on one node if it would fit into it.

Reply:
Ok.  Totally fine.  Maybe a little commentary in the comment for future developers would be helpful, maybe not.